src/client/room.js

```diff
@@ -961,11 +978,7 @@ function EditRoleMorph(room, role) {
     if (role.users.length) {  // occupied
         // owner can evict collaborators, collaborators can evict guests
 
-        // Check if we are occupying that role already. If not, give the
-        // option for collaborative editing
-        if (role.users.indexOf(this.room.myUserId()) === -1) {
```

Comment:
This was easier to just remove than update. I don't think there is a problem allowing the same user occupy the same role from the room editor

Reply:
Added the functionality back in a simpler way.